changelogs/fragments/216-virt-refactor.yml

```
@@ -0,0 +1,3 @@
+minor_changes:
+  - virt - refactored common libvirt functionality into module_utils to enable code reuse between modules.
```

```suggestion
minor_changes:
  - virt - refactored common libvirt functionality into module_utils to enable code reuse between modules.
```
If it doesn't change / break the current user interface, we can remove the fragment